### ISSUE TITLE
missing /var/log/mysql/ after 9ed73fd on mysql:5.7

### ISSUE THREAD

**Issue by hikaen2** [docker-library/mysql#908]
I'm using mysql:5.7 and outputting slow query log to /var/log/mysql/.
When I did a docker pull today, the /var/log/mysql directory was missing:

```
$ docker run mysql:5.7 ls -lF /var/log
total 8
-rw-r----- 1 mysql mysql    0 Jul 26 23:29 mysqld.log
-rw------- 1 root  root  6932 Jul 26 23:29 yum.log
```

I think this commit probably affected it: https://github.com/docker-library/mysql/commit/9ed73fdd2f2fd77591d0020e13c37602a0600eb8

For reference, mysql:5.7-debian has /var/log/mysql/:

```
$ docker run mysql:5.7-debian ls -lF /var/log
total 364
-rw-r--r-- 1 root  root   2200 Aug  2 04:50 alternatives.log
drwxr-xr-x 1 root  root   4096 Aug  2 04:50 apt/
-rw-rw---- 1 root  utmp      0 Aug  1 00:00 btmp
-rw-r--r-- 1 root  root  29531 Aug  2 04:50 dpkg.log
-rw-r--r-- 1 root  root  32000 Aug  2 04:49 faillog
-rw-rw-r-- 1 root  utmp 292000 Aug  2 04:49 lastlog
drwxr-x--- 2 mysql adm    4096 Aug  2 04:50 mysql/
-rw-rw-r-- 1 root  utmp      0 Aug  1 00:00 wtmp
```

The expected behavior is that /var/log/mysql/ does not disappear after docker pull mysql:5.7.

Or is there another appropriate directory to log to?

Thank you.

**Comment by tianon** [comment]
If you want that log to be persistent, you probably want to mount it to a directory or volume on the host. :see_no_evil:

I think the "proper" fix to this is probably what I outlined in https://github.com/docker-library/mysql/pull/797#issuecomment-967544899 :sweat_smile:

**Comment by hikaen2** [comment]
Thank you for your reply.
I don't want the log to be persist, because I'm only logging temporarily for development purposes.
So it is better if the log disappear when the container is restarted.

**Comment by tianon** [comment]
Now I can close this as fixed via #797 :eyes:
